change/@fluentui-react-62019bfd-162b-47e4-b2b6-cc3982f8f216.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

This is technically a minor change since it updates the API:

```suggestion
  "type": "minor",
```